test/data/test_builtin_datasets.py

```diff
@@ -89,9 +83,13 @@ def test_penntreebank(self):
         tokens_ids = [vocab[token] for token in 'the player characters rest'.split()]
         self.assertEqual(tokens_ids, [2, 2550, 3344, 1125])
 
+        # Add test for the subset of the standard datasets
+        train_dataset, test_dataset = PennTreebank(data_select=('train', 'test'))
+        train_iter, test_iter = torchtext.experimental.datasets.raw.PennTreebank(data_select=('train', 'test'))
```

Comment:
I'd check whether the returned iterator then is also indeed for training or testing as you'd expect. That way we make sure this subset selection is working.

Reply:
Fixed